build.yaml

```diff
@@ -1,8 +1,8 @@
 nodejs:
   - "0.10"
   - "0.12"
-  - "iojs-3"
-  - "4.0"
+  - "4.3"
```

Comment:
Nice, that would be nice to use so we don't have to constantly update.  I'll see if it's something we can handle in our jenkins env with nodeenv.